Flow.Launcher.Core/Plugin/PluginAssemblyLoader.cs

```diff
@@ -11,17 +11,22 @@ internal class PluginAssemblyLoader : AssemblyLoadContext
     {
         private readonly AssemblyDependencyResolver dependencyResolver;
 
-        private readonly AssemblyDependencyResolver referencedPluginPackageDependencyResolver;
+        private static readonly AssemblyDependencyResolver referencedPluginPackageDependencyResolver;
```

Comment:
why static?

Reply:
Because we create assembly loader for every plugin, but the referenced dependency are for Flow, which is the same across all instance, so no need for creating a new one for every instance.